### PR TITLE
Change target framework and remove no longer required dependencies

### LINKED ISSUE #79: Make ExcelAdapter always target netstandard2.0
#### Broken rules:

If possible, should make sure we only have one build target for Excel_Toolkit, which should be netstandard 2.0, and use for all builds. If works, should also remove additional ZCT build.

Whilst investigating this, it seems this means that the issue with saving larger files found and fixed by @pawelbaran in https://github.com/BHoM/Excel_UI/issues/330 is no longer an issue, and the code and dependencies added to remedy the issue can be removed.

#### Suggestions to restore compliance:

Change target framework and align.

### PR DESCRIPTION
**Pull request by IsakNaslundBh**
### NOTE: Depends on 

https://github.com/BHoM/BHoM_Engine/pull/3453

### Issues addressed by this PR

Closes #79

Changes the target framework of the debug and release build to simply be netstandard2.0 . 

Whilst attempting to do this, realised that the reason this toolkit was still on 4.7.2 net framework was the `VerifySecurityEvidenceForIsolatedStorage` to handle saving of large files. From testing with file provided in https://github.com/BHoM/Excel_UI/pull/331 this does not seem to be an issue any longer for netstandard2.0 , hence the method has been removed.

@pawelbaran would greatly appreciate if you are able to check that you are happy with this removal, given you raised this and fixed it in the first place.

Ofc general functionality of the toolkit should also be checked to ensure this does not have any other detrimental impacts on the toolkit.

### Test files

[Test file here](https://burohappold.sharepoint.com/:f:/r/sites/BHoM/02_Current/12_Scripts/02_Pull%20Request/BHoM/Excel_Toolkit/%2381-ChangeTargetFrameworkAndRemoveNoLongerRequiredDependencies?csf=1&web=1&e=KeYLfY)

### CODE REVIEW

**Comment by IsakNaslundBh** [comment]
@BHoMBot check compliance

**Comment by IsakNaslundBh** [comment]
@BHoMBot check project-compliance

**Review by pawelbaran — CHANGES_REQUESTED** [review]
Code changes make sense to me, but unfortunately Excel adapter does not work inside Revit on this PR, following exception is thrown:
image

Via trial and error I identified that it is `DocumentFormat.OpenXml.Packaging.OpenXmlPackage` type (a property of `DocumentFormat.OpenXml.Packaging.SpreadsheetDocument`) that cannot be loaded.

I was trying to track down the origins using Fusion Log, got this:
image

The above does not make much sense to me, why is `DocumentFormat.OpenXml` trying to load `System.IO.Packaging` 4.0.5, if it clearly depends on 6.0.0?
image

Meanwhile, I do not have any issues with loading types from `System.IO.Packaging` directly in Push method, e.g. adding `System.IO.Packaging.Package p = null` binds nicely to 6.0.0.

Going further, at the moment of calling `Push`, `DocumentFormat.OpenXml` 2.16 is already loaded into app domain from _C:\\ProgramData\\BHoM\\Assemblies\\DocumentFormat.OpenXml.dll_, which is alright.

So to summarise: everything seems to be perfectly fine, right assemblies being loaded and referenced, until `DocumentFormat.OpenXml.Packaging.OpenXmlPackage` is reflected, causing an attempt to load `System.IO.Packaging` 4.0.5 instead of 6.0.0. My gut feeling is that there is some voodoo happening due to the app domain being owned by Revit that sits on .NET Framework, but can't find out where & why. @adecler, any ideas what could be wrong here? Suggestions how to debug deeper?

**Comment by adecler** [comment]
@pawelbaran , my initial finding leads to SAP Toolkit being the potential problem as it still depends ClosedXML version 0.95:

image

**Comment by pawelbaran** [comment]
Thanks for help @adecler - unfortunately it does not fix Revit :< Just tried, still getting the same exception

**Comment by pawelbaran** [comment]
After the recent issue with `System.IO.Packaging` assembly raised to @michal-pekacki by the users, I had another think about fixing it - having succeeded with the initial testing, I rebased the branch and pushed the fix. Something to pick up early 8.1 I believe.

**Comment by pawelbaran** [comment]
I have just rebased the branch, @IsakNaslundBh could you please have a look at the code and confirm you have nothing against the changes as the originator of the PR? @michal-pekacki could you please test and see if it solves your issues? I will do the testing of Grasshopper interop.

**Comment by pawelbaran** [comment]
@BHoMBot check installer

**Comment by pawelbaran** [comment]
@BHoMBot check installer

**Review by michal-pekacki — DISMISSED** [review]
I tested on different Revit versions and it works without issues, happy to approve 👍

**Comment by pawelbaran** [comment]
@BHoMBot check required

**Comment by IsakNaslundBh** [comment]
@BHoMBot check project-compliance

**Comment by pawelbaran** [comment]
@BHoMBot check installer

**Review by michal-pekacki — APPROVED** [review]
Tested together with https://github.com/BHoM/BHoM_Engine/pull/3453 and it works, reapproving after changes 👍

**Review by pawelbaran — APPROVED** [review]
Approving following testing and a chat with @adecler and @IsakNaslundBh
